force-app/main/default/lwc/newProgramEngagement/__tests__/data/currentPageReference.json

```diff
@@ -0,0 +1,7 @@
+{
+    "type": "standard__navItemPage",
+    "attributes": {
+        "apiName": "Wire"
```

Comment:
This should probably mimic what is actually expected to be passed in.

Reply:
Ok. I looked at lwc-recipes and did similarly but will check it again.